test-packages/integration-tests/test/auth-flows/auth-flow.spec.ts

```diff
@@ -96,7 +101,7 @@ describe('OAuth flows', () => {
       destinationName: systems.s4.providerBasic,
       jwt: accessToken.provider
     });
-
+    assertHttpDestination(destination!);
```

Comment:
[q] So now when we use `getDestination`, we'll always have to call `assertHttpDestination` for http destinations, before calling execute?